tools/memap.py

```diff
@@ -620,14 +629,19 @@ def main():
         ", ".join(MemapParser.export_formats))
 
     parser.add_argument('-v', '--version', action='version', version=version)
+    
+    parser.add_argument('-d', '--detailed', action='store_true', help='Displays the elements in "Misc" in a detailed fashion', required=False)
 
     # Parse/run command
     if len(sys.argv) <= 1:
         parser.print_help()
         sys.exit(1)
 
 
-    args = parser.parse_args()
+    args, remainder = parser.parse_known_args()
```

Comment:
This one slipped trough (I was porting my changes from an older version of memap.py

I'll change it right away :)